build/common-replacements.js

```diff
@@ -26,3 +26,16 @@ module.exports.altIndexOfUseReplacement = [
     /(\W)([\w\.\(\),\[\]]+)(\.indexOf\()/gm
   , '$1indexOf($2, '
 ]
+module.exports.objectKeysDefine = [
```

Comment:
Any of [these browsers](https://cloud.githubusercontent.com/assets/1128607/8131283/3a814a02-10e7-11e5-85c0-8d3887303f70.png)?

As far as I could tell on https://kangax.github.io/compat-table/es5/,  all the above browsers supported `Object.keys`, only if we go as far back as `IE8`, `FF 3.5` or `Safari 4`, does that actually become an issue.